net-analyzer/gvm-tools/gvm-tools-2.0.0.ebuild

```diff
@@ -17,20 +17,24 @@ KEYWORDS="~amd64 ~x86"
 IUSE=""
 
 RDEPEND="
-	dev-python/defusedxml[${PYTHON_USEDEP}]
-	dev-python/lxml[${PYTHON_USEDEP}]
-	dev-python/paramiko[${PYTHON_USEDEP}]
-	dev-python/pythondialog:0[${PYTHON_USEDEP}]
-	dev-python/setuptools[${PYTHON_USEDEP}]
+	net-analyzer/python-gvm[${PYTHON_USEDEP}]
 	!net-analyzer/openvas-cli
 	!net-analyzer/openvas-tools"
 
 DEPEND="
 	${RDEPEND}"
 
+distutils_enable_tests unittest
+
 src_prepare() {
 	distutils-r1_python_prepare_all
```

Comment:
You should call `distutils-r1_<phase-function>`, the same as phase you're writing.